04_setup_ironic.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -ex
+
+source common.sh
+
+# Get the various images
+source get_images.sh
+
+# ironic dnsmasq and ipxe config
+cp ironic/dnsmasq.conf $IRONIC_DATA_DIR
+cp ironic/dualboot.ipxe ironic/inspector.ipxe $IRONIC_DATA_DIR/html
+
+# Now that we have the Environment and the image, we can pull the image and start the ironic service
+sudo podman pull "$IRONIC_IMAGE"
+sudo podman pull "$IRONIC_INSPECTOR_IMAGE"
+
+#if [ ! -e /etc/sysconfig/network-scripts/ifcfg-brovc ] ; then
+#    echo -e "DEVICE=brovc\nONBOOT=yes\nTYPE=Ethernet\nBOOTPROTO=static\nIPADDR=172.22.0.1\nNETMASK=255.255.255.0" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-brovc
+#    ifdown brovc || true
+#    ifup brovc
+#fi
+
+# Workaround so that the dracut network module does dhcp on eth0 & eth1
+if [ ! -e $IRONIC_DATA_DIR/html/images/redhat-coreos-maipo-47.284-openstack_dualdhcp.qcow2 ] ; then
+    pushd $IRONIC_DATA_DIR/html/images
+    qemu-img convert redhat-coreos-maipo-47.284-openstack.qcow2 redhat-coreos-maipo-47.284-openstack.raw
```

Comment:
nit: will `qemu-img` fail if `redhat-coreos-maipo-47.284-openstack.raw` accidentally exists from some unsuccessful previous run?

Reply:
Nope, its ok if the file already exists